cmd/root.go

```diff
@@ -73,3 +85,11 @@ func Execute() error {
 	}
 	return nil
 }
+
+func PrintError(err error) {
+	os.Stderr.WriteString(fmt.Sprintf("Error: %s\n", err.Error()))
+}
+
+func printOutput(output string) {
+	os.Stdout.WriteString(fmt.Sprintf("%s\n", output))
```

Comment:
is this because the linter complains about fmt.Println? maybe we can add exception if so

Reply:
Let's keep it as it is clearly shows where the output goes between PrintOutput and PrintError. Turns out, logrus' default output goes to stderr...